src/internal/pachd/builder.go

```diff
@@ -283,6 +286,22 @@ func (b *builder) registerPFSServer(ctx context.Context) error {
 	return nil
 }
 
+func (b *builder) registerPJSServer(ctx context.Context) error {
+	apiServer := pjs_server.NewAPIServer(pjs_server.Env{
+		DB:               b.env.GetDBClient(),
+		GetPermissionser: b.env.AuthServer(),
+		GetAuthToken:     auth.GetAuthToken,
+		// TODO: figure out how to set this.
+		GetStorageClient: func(ctx context.Context) storage.FilesetClient {
```

Comment:
Why do we need a callback here?  This can just be `storage.FilesetClient`.

Reply:
Agree.  Many clients close over ctx, but fortunately not the FilesetClient.

Reply:
Though since the client doesn't have a conn this early, you probably have to do it like this.

Reply:
Bryce's PR will actually change this to a StorageServer, so when that gets merged in, he or I will have to update this anyways 😆